packages/api/internal/handlers/template_build_logs.go

```diff
@@ -75,13 +81,23 @@ func (a *APIStore) GetTemplatesTemplateIDBuildsBuildIDStatus(c *gin.Context, tem
 	if team == nil {
 		msg := fmt.Errorf("user doesn't have access to env '%s'", templateID)
 		telemetry.ReportError(ctx, msg)
-
 		a.sendAPIStoreError(c, http.StatusForbidden, fmt.Sprintf("You don't have access to this sandbox template (%s)", templateID))
-
 		return
 	}
 
-	status := dockerBuild.GetStatus()
+	buildDB, err := a.db.GetEnvBuild(ctx, buildUUID)
```

Comment:
I think this may not be needed, you could invalidate cache from `BuildStatusSync`, but let's keep it simple for now